version.properties

```diff
@@ -1,9 +1,9 @@
 #Mon, 30 Aug 2021 11:48:28 +0200
 
 # Version Information for Vanilla / Release builds
-versionName=18.3-rc-1
-versionCode=1111
+versionName=18.3-rc-2
+versionCode=1114
 
 # Version Information for other flavors: zalpha, wasabi, jalapeno
-alpha.versionName=alpha-320
-alpha.versionCode=1112
+alpha.versionName=alpha-321
+alpha.versionCode=1115
```

Comment:
Version Codes manually bumped (via 3964aa33ccf52b5236e32057d76c252bd58d2b9f) as the automatic bump was otherwise (expectedly) incorrect after the 18.2.1 hotfix that was done between rc-1 and rc-2.